dipy/data/fetcher.py

```diff
@@ -611,33 +635,22 @@ def read_mni_template(contrast="T2"):
     >>> T2_nifti = read_mni_template("T2") # doctest: +SKIP
     Get both files in this order:
     >>> T1_nifti, T2_nifti = read_mni_template(["T1", "T2"]) # doctest: +SKIP
-
     """
-    folder = pjoin(dipy_home, 'mni_template')
+    files, folder = fetch_mni_template()
     file_dict = {"T1":pjoin(folder, 'mni_icbm152_t1_tal_nlin_asym_09a.nii'),
```

Comment:
(that might also be the reason we are using 'a')

On Thu, Aug 6, 2015 at 10:20 AM, Ariel Rokem <arokem@gmail.com> wrote:

> No rush. The ROIs that we are using for tractography segmentation are all
> based on the MNI template anyway.
>
> On Thu, Aug 6, 2015 at 10:18 AM, Samuel St-Jean <notifications@github.com>
> wrote:
>
>> In dipy/data/fetcher.py
>> <https://github.com/nipy/dipy/pull/681#discussion_r36440520>:
>>
>> >      """
>> > -    folder = pjoin(dipy_home, 'mni_template')
>> > +    files, folder = fetch_mni_template()
>> >      file_dict = {"T1":pjoin(folder, 'mni_icbm152_t1_tal_nlin_asym_09a.nii'),
>>
>> They also have a seperate license for commercial and non commercial
>> usage, so maybe you can just point a download link for people while the
>> re-licensing happens, so that people might have it in the meantime if it's
>> that important.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/nipy/dipy/pull/681/files#r36440520>.
>>
>
>